### PR TITLE
Hparkersd patch 1

### PR DESCRIPTION
**Pull request by hparkersd**
Updates related to teams

### CODE REVIEW

**Review by n2ygk — APPROVED** [review]

**Comment by n2ygk** [comment]
@hparkersd is this still a draft? Please remove the draft status if you are ready

**Comment by hparkersd** [comment]
> @hparkersd is this still a draft? Please remove the draft status if you are ready

I am still working on it.  I needed to stop for the day.

**Comment by hparkersd** [comment]
I am ready to go.  Will refrain from tinkering with it.

**Review by n2ygk — CHANGES_REQUESTED** [review]
I did a pretty cursory review this time. Perhaps ask one or two others to look at the content on the [draft site](https://deploy-preview-74--examtools-docs.netlify.app/).

File: content/docs/cve/Zoom.md [hunk 1]
```diff
@@ -130,104 +128,101 @@ In [Personal/Meetings](https://zoom.us/meeting) settings click [Schedule a Meeti
...
+When using a second device (usually a phone), applicants frequently connect audio, which leads to audio feedback
+from the speaker of the phone into the microphone of the main computer. Have the applicant click on `... More`
 near the bottom right and then `Disconnect Audio`.
 
 ![Zoom iOS More...](../images/zoom_ios_ellipsis.png)
 
 ![Zoom iOS Disconnect Audio](../images/zoom_ios_disconnect_audio.png)
+
+## Recording with Zoom
+
```
Comment (n2ygk) [inline content/docs/cve/Zoom.md:216]:
```suggestion
The ARRL/VEC and others have asked that we not record candidate sessions. If you choose to record, be certain to *never record personally-identifiable information (PII)* such as photo ID and only record if the applicant has given explicit permission to do so.
```

File: content/docs/cve/cveTeams.md [hunk 2]
```diff
@@ -16,14 +16,14 @@ type: book  # Do not modify.
 
 ## Intro:
```
Comment (n2ygk) [inline content/docs/cve/cveTeams.md:17]:
```suggestion
## Introduction
```

File: content/docs/cve/cveTeams.md [hunk 3]
```diff
@@ -16,14 +16,14 @@ type: book  # Do not modify.
...
 
-The primary purpose of a "team account" in ExamTools is to own exam sessions and schedules. It's purely an administrative entity. Team accounts can have a name, contact email, and website. There is no need to add everyone who is on your actual team to the ExamTools "team account".
+The primary purpose of a "team account" in ExamTools is to own exam sessions and schedules. It's purely an administrative entity. Team accounts can have a name, contact email, and website. There is no need to add every Volunteer Examiner (VE) who may attended attend an exam to the ExamTools "team account".  Only VEs with team lead permissions from their VEC will be able to create a team in ExamTools.  If you are a GLAARG session manager, W5YI CVE or a SANDARC team lead, your VEC will update ExamTools automatically. If you are an ARRL team lead, please send the emailed confirmation of your approval from Maria Somma ARRL-VEC to verifications@examtools.org. 
 
 ## Create a New Team Account:
```
Comment (n2ygk) [inline content/docs/cve/cveTeams.md:21]:
```suggestion
## Create a New Team Account
```

File: content/docs/cve/cveVEList.md [hunk 4]
```diff
@@ -14,20 +14,22 @@ title: Edit the VE List in an ExamTools Session
 type: book  # Do not modify.
 ---
 
-## Add a VE to an Exam Session:
+## Add a Volunteer Examiner (VE) to an Exam Session:
```
Comment (n2ygk) [inline content/docs/cve/cveVEList.md:17]:
```suggestion
## Add a Volunteer Examiner (VE) to an Exam Session
```

File: content/docs/cve/cveVEList.md [hunk 5]
```diff
@@ -14,20 +14,22 @@ title: Edit the VE List in an ExamTools Session
...
 
 * **ExamTools (ET)** should open in the **[Session List](https://exam.tools/ve/sessions)**.
 
-To add a VE to an exam session, select an existing session from the **Session List**. 
+To add a VE to an exam session, select a session from the **Session List**. 
 
 By default, the Applicant tab is active.  Click on the Volunteer Examiner tab to the left of the active tab.  At the top of the VE page, located and click the button **+ ADD VES**.
 
+> **Required on a New Session Listing:**  If you see "You are not part of this session" on a gold alert ribbon when you first access the session listing, make sure to click the **JOIN** button from the Applicant Tab. Next, click on the Volunteer Examiner tab. If fail to join the session, you will see that the session has 0 VEs and a blue box with "No Volunteer Examiners have been added to the session". Return back to the gold ribbon on the Applicant Tab and click the **JOIN** button. Then, click on the Volunteer Examiner tab.  You must then change your permissions to co-owner in order to add the rest of your team.
```
Comment (n2ygk) [inline content/docs/cve/cveVEList.md:31]: Did you intend this to be a quoted block?
```suggestion
**Required on a New Session Listing:**  If you see "You are not part of this session" on a gold alert ribbon when you first access the session listing, make sure to click the **JOIN** button from the Applicant Tab. Next, click on the Volunteer Examiner tab. If fail to join the session, you will see that the session has 0 VEs and a blue box with "No Volunteer Examiners have been added to the session". Return back to the gold ribbon on the Applicant Tab and click the **JOIN** button. Then, click on the Volunteer Examiner tab.  You must then change your permissions to co-owner in order to add the rest of your team.
```

File: content/docs/cve/printedExamCreation.md [hunk 6]
```diff
@@ -14,12 +14,16 @@ title: Print and Grade Hard Copy Exams in ExamTools
 type: book  # Do not modify.
 ---
 
+## Required to Print Exams
+
+To generate and print exams in ExamTools, you must have team lead permissions or be a Volunteer Examiner (VE) on a team with Print Exam permissions. Some teams leads may need to contact their VEC to update their status. Make sure to check out the [New Team Lead Information]({{<relref "cveNewTeamLead.md">}}), if you do not see Answer Keys and Generate Exam in the left navigation of ExamTools.
+  
 ## Printing Exam Sheets for Manually Graded Exams
 
 Follow these instructions to print out randomly generated exams for use with a manual grading key
```
Comment (n2ygk) [inline content/docs/cve/printedExamCreation.md:23]:
```suggestion
Follow these instructions to print out randomly generated exams for use with a manual grading key:
```

File: content/docs/cve/printedExamCreation.md [hunk 7]
```diff
@@ -14,12 +14,16 @@ title: Print and Grade Hard Copy Exams in ExamTools
...
 
-* Log into [ExamTools](https://exam.tools) software using your user name and password
-* On the left hand navigation bar, click **Generate Exam**
+* Log into [ExamTools](https://exam.tools) using your user name and password
+* On the left navigation bar, click **Generate Exam**
 ![Generate Exams](../images/generateExams.png)
 
 * The default selection is **Manual Grading**, indicated by a grey highlight
```
Comment (n2ygk) [inline content/docs/cve/printedExamCreation.md:29]:
```suggestion
* The default selection is **Manual Grading**, indicated by a grey highlight.
```

**Review by n2ygk — CHANGES_REQUESTED** [review]
Wow, @hparkersd, a ton of improvements. I've made a few suggestions which you can "add to batch" and then apply if you agree with them. Some where stuff you didn't touch on this time around but that I noticed while viewing the rendered docs (at https://deploy-preview-74--examtools-docs.netlify.app/docs/).

**Pull request merged**